rtp.h

```diff
@@ -107,7 +107,7 @@ janus_audiocodec janus_audiocodec_from_name(const char *name);
 int janus_audiocodec_pt(janus_audiocodec acodec);
 
 typedef enum janus_videocodec {
-	JANUS_VIDEOCODEC_NONE,
+	JANUS_VIDEOCODEC_NONE = 0,
```

Comment:
These changes are unrelated and belong in a different commit.

Reply:
Do you mean that I should create a separate commit/PR?

Reply:
Yeah, this change is basically cosmetic since `JANUS_VIDEOCODEC_NONE` is already defined as 0 (because of C's rules for enums), and is unrelated to your `g_malloc0` changes, so should be in a separate commit.

Reply:
It is, but someone can change it to e.g. -2. There are such enums in Janus. I make it start from 0 just to show it must start from 0 and remove code something like `some field = JANUS_VIDEOCODEC_NONE ` because of g_malloc0.